docs/book/index.md

```diff
@@ -1,11 +1,10 @@
 > ### Deprecated!
 > 
 > Due to the amount of integration required to support console tooling via the
-> MVC, and because [better, more standalone solutions
-> exist](https://github.com/zfcampus/zf-console), **we will not be maintaining
+> MVC, and because better, more standalone solutions exist, **we will not be maintaining
 > laminas-mvc-console long term**. We strongly urge developers to start migrating their
 > MVC-based console tooling to use other libraries, such as
-> [zf-console](https://github.com/zfcampus/zf-console).
+> [symfony/console](https://github.com/symfony/console).
```

Comment:
> Recommend symfony/console instead.

And why not [laminas-cli](https://github.com/laminas/laminas-cli)? 😉

Reply:
Updated.